### PR TITLE
arpeggiate support added

### LINKED ISSUE #64: Missing: Arpegios

### PR DESCRIPTION
**Pull request by rvilarl**
fixes #64

image

### CODE REVIEW

**Comment by rvilarl** [comment]
@jaredjj3 hope you like it :)

**Comment by jaredjj3** [comment]
It looks great! Thanks for your patience, I'll try to review soon.

File: src/types.ts [hunk 1]
```diff
@@ -53,6 +53,8 @@ export type NoteMessage = {
   duration?: number;
   grace: boolean;
   graceSlash: boolean;
+  arpeggiate: boolean;
+  arpeggiateDirection?: string;
```
Comment (jaredjj3) [inline src/types.ts:57]: This is fine for now, but we should start thinking about breaking up note messages into different subtypes. Perhaps like choices in the content of the [`<note>`](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/note/) element.

**Review by jaredjj3 — APPROVED** [review]
Looks good! I'm pleasantly surprised by how simple this PR was. It's a sign we're moving in the right direction.

File: src/renderer.ts [hunk 2]
```diff
@@ -184,6 +184,20 @@ export class Renderer implements VexmlMessageReceiver {
             );
             t.expression(() => (graceStart = -1));
           }
+          // Process arpeggiatte
+          if (message.arpeggiate) {
+            switch (message.arpeggiateDirection) {
+              case undefined:
```
[the remark this answers is not in the document]
Comment (rvilarl) [reply src/renderer.ts]: Done

File: src/types.ts (hunk 1 above)
Comment (rvilarl) [reply src/types.ts:57]: agree

**Pull request merged**